lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -103,6 +103,7 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
   else {
     // Output reportback images with date last updated.
     $form['reportback_submissions']['reportback_item']['#attributes']['is_supersized'] = FALSE;
+    $form['reportback_submissions']['reportback_item']['#attributes']['data-validate'] = '';
```

Comment:
wow `unset` duhhhh. that's why it was being funky.

Previously `data-validate` was using [`hasImage`](https://github.com/DoSomething/phoenix/blob/7b2a84aa399c9bd2eb9a91ddc6ad60bf50e5ccd1/lib/themes/dosomething/paraneue_dosomething/js/validators/reportback.js#L34-L48) whose description is `make sure an image is uploaded`, so I think that's all that this part was handling?